### ISSUE TITLE
Theme Jonatan broken

### ISSUE THREAD

**Issue by toperx** [ohmyzsh/ohmyzsh#10513]
### Describe the bug

Afther last update in all my devices theme Jonatan broken

### Steps to reproduce

Update to last version omz
Change theme to Jonatan
Run zsh

### Expected behavior

```
┌─(~)──────────────────────────────────────────────────────────────────────────(pbbushuev@BUSHUEV-ROSA:pts/0)─┐
└─(16:30:29)──> bash                                                                                                                                                                                                                                                       ──(Чт,дек16)─┘
[pbbushuev@BUSHUEV-ROSA ~]$ zsh
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:40: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:41: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:40: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:41: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:40: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:41: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:40: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:41: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:40: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:41: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:40: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:41: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:40: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:41: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:40: bad option: -v
/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh/themes/jonathan.zsh-theme:41: bad option: -v
┌─(~)──────────────────────────────────────────────────────────────────────────(pbbushuev@BUSHUEV-ROSA:pts/0)─┐
└─(16:39:23)──> upgrade_oh_my_zsh                                                                                                                                                                                                                                          ──(Чт,дек16)─┘
Note: `upgrade_oh_my_zsh` is deprecated. Use `omz update` instead.
Updating Oh My Zsh
From https://github.com/ohmyzsh/ohmyzsh
 * branch            master     -> FETCH_HEAD
Current branch master is up to date.
```

### Screenshots and recordings

![Снимок экрана в 2021-12-16 16-58-57](https://user-images.githubusercontent.com/25674025/146316763-bce77157-3eb5-4420-9364-c279de27a85c.png)


### OS / Linux distribution

Rosa Linux Enterprise Desktop

### Zsh version

5.0.2

### Oh My Zsh version

master (5403db8)

### Terminal emulator

Mate terminal 1.16.3

### If using WSL on Windows, which version of WSL

_No response_

### Additional context

~/.zshrc

export ZSH="/home/pbbushuev@y-s.shelf-dobycha.local/.oh-my-zsh"
ZSH_THEME="jonathan"
plugins=(git tmux command-not-found shell-proxy sudo yum)
source $ZSH/oh-my-zsh.sh

**Comment by mcornella** [comment]
Hey @toperx, try updating again with `omz update`, this was fixed in a later commit.

**Comment by mcornella** [comment]
Fixed in 7d03ea18 and 41c15a21.

**Comment by toperx** [comment]
Thx! All good!
